packages/store/src/workspace/page.ts

```diff
@@ -279,6 +279,36 @@ export class Page extends Space<FlatBlockMap> {
     });
   }
 
+  updateSelectedColumnValue(
+    rowId: string,
+    columnId: string,
+    oldValue: string,
+    value?: string
+  ) {
+    this.transact(() => {
+      const yColumns = this.yColumns.get(rowId);
+      assertExists(yColumns);
+      const cell = yColumns.get(columnId) as Y.Map<string[]> | undefined;
+      if (!cell) return;
+
+      const selected = cell.get('value') as string[];
+      let newSelected = [...selected];
+      if (value !== undefined) {
+        // rename tag
```

Comment:
I believe mixing rename and deleting actions based on `value` param is an anti-pattern